bitnami/keycloak/templates/NOTES.txt

```diff
@@ -18,6 +18,14 @@ To access Keycloak from outside the cluster execute the following commands:
    echo "Keycloak URL: http{{ if .Values.ingress.tls }}s{{ end }}://{{ (tpl .Values.ingress.hostname .) }}/"
    echo "$CLUSTER_IP  {{ (tpl .Values.ingress.hostname .) }}" | sudo tee -a /etc/hosts
 
+{{- if .Values.adminIngress.enabled }}
+Provided that the environment variable `KC_HOSTNAME_ADMIN` has been set, as well as the `frontendUrl` property of
+the `master` realm (see README for an example) :
+
```

Comment:
```suggestion
The admin area of Keycloak has been configured to point to a different domain ({{ .Values.adminIngress.hostname }}). Please, remember to  update the `frontendUrl` property of the `master` (or any other) realm for it to work properly (see README for an example) :
```

Reply:
I'm not saying this is the right wording, but I think something like it can make it 😁

Reply:
Thanks for the suggestion, I've included it 👍